README.md

```diff
@@ -47,27 +47,55 @@ Default: `local.net/nfs`
 
 Default: `local-nfs`
 
-### `hostPath`
+### `provisionerVolume`
 
-If this is empty, no local storage will be used, making this completely emphemeral.
+There are a number of modes you can use. These are the available options.
```

Comment:
I wanted to make this a bit more declarative. In the previous implementation, there were certain configurations that we're super-obvious to me. For instance, if hostPath was not set, it used emptyDir. Or, setting useTmpFs while defining other variables made the setting do nothing. 

By using one "mode" value, it makes it very obvious which mode you're using.